doc/content/bib/blackbear.bib

```diff
@@ -244,3 +244,50 @@ @article{poyet2009temperature
 	  year={2009},
 	  publisher={Elsevier}
 	}
+
+	@article{lee1998plastic,
+  title={Plastic-damage model for cyclic loading of concrete structures},
+  author={Lee, Jeeho and Fenves, Gregory L},
+  journal={Journal of engineering mechanics},
+  volume={124},
+  number={8},
+  pages={892--900},
+  year={1998},
+  publisher={American Society of Civil Engineers}
+}
+
+@book{lee1996theory,
```

Comment:
Change the following:
`@book` to `@phdthesis`
`@publisher` to `@school`

Reply:
Edited